boards.txt

```diff
@@ -350,3 +423,58 @@ OSHChip.menu.softdevice.s130.softdeviceversion=2.0.1
 OSHChip.menu.softdevice.s130.upload.maximum_size=151552
 OSHChip.menu.softdevice.s130.build.extra_flags=-DNRF51 -DS130 -DNRF51_S130
 OSHChip.menu.softdevice.s130.build.ldscript=armgcc_s130_nrf51822_xxac.ld
+
+
+PCA1000X.name=PCA1000X
```

Comment:
Is this a Option Rename this to "PCA1000X (nRF51)" to Group PCA nRF51 and nRF52 Boards.